specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/examples/securitydomainoperationstatus-example.json

```diff
@@ -5,10 +5,6 @@
   },
   "responses": {
     "200": {
-      "headers": {
```

Comment:
Why remove this instead of just making it an LRO? The "pending" end point would work well. In the client SDK, we'll end up making these "pseudo-LROs" anyway, so having that support automatically from swagger via our generators would save us a lot of time.